### ISSUE TITLE
Solarium\QueryType\Select\Query\Component\Grouping::FORMAT_SIMPLE Does Not Work

### ISSUE THREAD

**Issue by bmeynell** [solariumphp/solarium#260]
``` php
$groupComponent = $select->getGrouping();                                  
$groupComponent->addField('fieldName');                                                             
$groupComponent->setLimit($limit);
$groupComponent->setFormat(\Solarium\QueryType\Select\Query\Component\Grouping::FORMAT_SIMPLE); 
```

Results in:

```
Notice: Undefined index: groups in vendor/solarium/solarium/library/Solarium/QueryType/Select/ResponseParser/Component/Grouping.php on line 77

Warning: Invalid argument supplied for foreach() in vendor/solarium/solarium/library/Solarium/QueryType/Select/ResponseParser/Component/Grouping.php on line 77
```

**Comment by chadicus** [comment]
Just tried this approach and the resulting data is not accurate. The `ngroups` value is not returned and therefore you cannot derive the total number of groups.

**Comment by mkalkbrenner** [comment]
Fixed by #468
